README.md

```diff
@@ -141,3 +141,31 @@ file `~/.oauth2l.token`.
 ```
 $ oauth2l reset
 ```
+
+## Options
+
+### `--json`
+
+Use a JSON key file downloaded from
+[Google Cloud Console](https://console.cloud.google.com/).
+
+```
+$ oauth2l fetch --json ~/service_account.json cloud-platform
+```
+
+### `--sso` and `--sso_cli`
+
+Use an SSO CLI to fetch OAuth token. The SSO CLI outputs an
```

Comment:
an single sign-on (SSO) command to .... The command outputs an OAuth access token to its stdout. The default command is for Google's corporate SSO.